charts/consul/values.yaml

```diff
@@ -2272,7 +2272,7 @@ connectInject:
     # If true, then all traffic redirection setup uses the consul-cni plugin.
     # Requires connectInject.enabled to also be true.
     # @type: boolean
-    enabled: false
+    enabled: true
```

Comment:
Any reason to enable CNI?

Reply:
nope, it was for local testing, reverting